src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -919,6 +928,17 @@ protected DynamoModel(IStartConfiguration config)
             LibraryServices.MessageLogged += LogMessage;
             LibraryServices.LibraryLoaded += LibraryLoaded;
 
+            string indexPath = Path.Combine(Environment.CurrentDirectory, "Index");
+            indexDir = FSDirectory.Open(indexPath);
```

Comment:
@reddyashish @RobertGlobant20 This is where we define search index file location, we want to discuss other than Dynamo install path, where to put it